src/ruleset-base.xml

```diff
@@ -42,6 +42,9 @@
 		<exclude name="Squiz.Arrays.ArrayDeclaration.NoComma"/>
 		<exclude name="Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast"/>
 	</rule>
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
+		<type>warning</type>
```

Comment:
Is there any reason to lower severity from error to warning?

Reply:
Updated